developers/roadmap.md

```diff
@@ -37,6 +41,9 @@ specification (as well as the caveats):
         * Does not support directly nested models
     * 3.0.0 (**released**): libsdformat9, SDFormat 1.7, but only a subset:
         * Does not support directly nested models
+    * 4.0.0 (**planned**): libsdformat10, SDFormat 1.7, but only a subset:
```

Comment:
How about changing this section from "Ignition Gazebo" to "Ignition" and the versions from numbers to collection names? Other libraries like Ignition Physics also directly depend on SDF.

Reply:
db3a35a3d45bdc1108aeda2f4c53d0a05bb54321